### ISSUE TITLE
Request to Enhance Search Functionality to Handle Both Place Names and DIGIPINs 

### ISSUE THREAD

**Issue by ronitjadhav** [kutazama/digipin-app#4]
What do you think of adding an enhancement to the current search functionality in the DigiPinMap component to handle both place names and DIGIPINs using a single input field?

**Comment by kutazama** [comment]
Good idea - though the code to calculate digipins was updated in https://github.com/kutazama/digipin-app/pull/3. Please check against the updated code, and submit a new PR.

**Comment by ronitjadhav** [comment]
I have rebased my PR and updated the function to retrieve the latitude and longitude value from Digipin, incorporating the new logic for generating Digipin.
